portfolio/src/main/webapp/homepage.js

```diff
@@ -54,6 +54,27 @@ async function loadComments() {
 
 $(document).ready(loadComments);
 
+/**
+ * Load comment form based on login status.
+ */
+async function loadCommentForm() {
+
+    // Request user information from server.
+    const responseJSON = await fetch("/user");
```

Comment:
Same as above. But if these parameters are for query parameters, then maybe Constants.QUERY_PARAM_USER.